libnopegl/src/backends/vk/rendertarget_vk.c

```diff
@@ -369,7 +369,7 @@ VkResult ngli_rendertarget_vk_init(struct rendertarget *s, const struct renderta
     const VkFramebufferCreateInfo framebuffer_create_info = {
         .sType           = VK_STRUCTURE_TYPE_FRAMEBUFFER_CREATE_INFO,
         .renderPass      = s_priv->render_pass,
-        .attachmentCount = s_priv->nb_attachments,
+        .attachmentCount = (uint32_t)s_priv->nb_attachments,
```

Comment:
What's the benefit of using size_t instead of the native type (uint32_t) ?

Reply:
Deferencing `attachments` won't involve a sign-extend. Unless you propose to use a `size_t` iterator on `i` but compare it to a `u32 nb_attachments` (which I'm not sure I like much). We can of course ignore that and use `u32` here.

Reply:
I'm in favor of using u32 (declaration and iterator) if that's ok with you.

Reply:
Alright, done